app/assets/javascripts/src/Metamaps.GlobalUI.js

```diff
@@ -1,4 +1,4 @@
-var Metamaps = {}; // this variable declaration defines a Javascript object that will contain all the variables and functions used by us, broken down into 'sub-modules' that look something like this
+var Metamaps = window.Metamaps || {}; // this variable declaration defines a Javascript object that will contain all the variables and functions used by us, broken down into 'sub-modules' that look something like this
```

Comment:
@devvmh based on the order that these files are getting required into application.js without this line we override the Metamaps.ReactComponents object that we created before and lose access to it.